flink-core/src/main/java/org/apache/flink/core/execution/DefaultExecutorServiceLoader.java

```diff
@@ -48,7 +52,7 @@
 	public static final DefaultExecutorServiceLoader INSTANCE = new DefaultExecutorServiceLoader();
 
 	@Override
-	public ExecutorFactory getExecutorFactory(final Configuration configuration) {
+	public ExecutorFactory getExecutorFactory(@Nonnull final Configuration configuration) {
```

Comment:
IIRC the community consensus is that all parameters are implicit non-null. FYI our [code style guide](https://flink.apache.org/contributing/code-style-and-quality-common.html#nullability-of-the-mutable-parts).